WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -453,6 +459,13 @@ import WordPressShared
         displayResultsStatus()
     }
 
+    private func displayNoConnectionView(_ topic: ReaderAbstractTopic) {
+        resultsStatusView.titleText = ReaderStreamViewController.responseForNoResults(topic).title
```

Comment:
I think it will be clearer if we use "Unable to Sync" or something similar as the title as we do in the alert case. "No likes yet" or "No recent posts" doesn't feel accurate and I tend to not read the subtitle line which is what actually contains the information of what's going on.